tests/Pest.php

```diff
@@ -62,6 +73,25 @@ enum Role
     case GUEST;
 }
 
+#[Meta([IsActive::class])]
+enum ReferenceType: int
+{
+    use InvokableCases, Options, Names, Values, From, Metadata, Comparable;
+
+    #[IsActive(true)]
+    case ACTIVE_TYPE = 1;
+
+    case INACTIVE_TYPE = 0;
+}
+    
+#[Meta([Desc::class])] // array
```

Comment:
Thanks for the PR! I agree with adding this feature and the PR looks good — fully backwards compatible and includes tests. Just wondering what this comment signifies?

Reply:
Ah it was a copy/paste from another test enum, I can clean it up

Reply:
@stancl should be set